examples/cocoa/CocoaAppDelegate.mm

```diff
@@ -57,7 +54,7 @@
     sf::RenderWindow      renderWindow;
     sf::Font              font{sf::Font::loadFromFile(resPath / "tuffy.ttf").value()};
     sf::Text              text{font};
-    sf::Texture           logo;
+    sf::Texture           logo{sf::Texture::loadFromFile(resPath / "logo.png").value()};
```

Comment:
Ditto about behavioral changes, we also lose the `NSLog` call here. Note that we already changed behavior in line 55 for the font, so I'm not sure...

Reply:
To be honest my attitude here was "meh whatever" about removing those logs. I'm not married to this solution and can go back to the previous solution if you feel strongly about keeping that.

Reply:
I never used/targeted Cocoa, I have no idea if those logs provide any real value or not, but honestly your solution is simpler and probably good enough. Unless you know anyone who targets Cocoa that can provide more feedback, I don't feel strongly about keeping the logs at all.